pkg/tfbridge/walk.go

```diff
@@ -114,8 +115,117 @@ func propertyPathToSchemaPathInner(
 	return nil
 }
 
+// Translate a a bridged provider schema path into a Pulumi property path.
+//
+// The function hides the complexity of mapping Terraform names to Pulumi property names,
+// joining Schema with user overrides in [SchemaInfo]s, and accounting for MaxItems=1
+// situations where Pulumi flattens collections to plain values.  and therefore SchemaPath
```

Comment:
" to plain values.  and therefore SchemaPath" - nit something's a bit odd with grammar there, do we need a "."?